### PR TITLE
camera_common: rename create_thumbnail to publish_thumbnail

### PR DESCRIPTION
**Pull request by deanlee**
rename create_thumbnail to publish_thumbnail, and send` PubMaster*` to it instead of `MultiCameraState*`

### CODE REVIEW

File: selfdrive/camerad/cameras/camera_common.cc [hunk 1]
```diff
@@ -295,9 +295,7 @@ static void create_thumbnail(MultiCameraState *s, const CameraBuf *b) {
   thumbnaild.setTimestampEof(b->cur_frame_data.timestamp_eof);
   thumbnaild.setThumbnail(kj::arrayPtr((const uint8_t*)thumbnail_buffer, thumbnail_len));
 
-  if (s->pm != NULL) {
```
Comment (pd0wm) [inline selfdrive/camerad/cameras/camera_common.cc:298]: Why is the null check no longer necessary? Is the pubmaster initialized on all camera types?

Comment (deanlee) [reply selfdrive/camerad/cameras/camera_common.cc:298]: yes,except for the camera_frame_stream that does not call `processing_thread`

Comment (pd0wm) [reply selfdrive/camerad/cameras/camera_common.cc:298]: Shall we just leave the null check just for extra safety? Don't want to have unexpected surprises if we refactor the code later.

Comment (deanlee) [reply selfdrive/camerad/cameras/camera_common.cc:298]: how about check it before call publish_thumbnail

Comment (pd0wm) [reply selfdrive/camerad/cameras/camera_common.cc:298]: that's fine too!

**Pull request merged**